src/wasm/literal.cpp

```diff
@@ -384,6 +386,10 @@ std::ostream& operator<<(std::ostream& o, Literal literal) {
         o << "exnref(" << literal.getExceptionPackage() << ")";
       }
       break;
+    case Type::anyref:
+      assert(literal.isNull() && "TODO: non-null anyref values");
+      o << "anyref(null)";
+      break;
     case Type::externref:
       assert(literal.isNull() && "TODO: non-null externref values");
       o << "externref(null)";
```

Comment:
In which case do we need non-null `anyref` and `externref` literals?

Reply:
This came up earlier in that these are currently not supported by `Literal`, which only can represent `null`s of any reference type, but Thomas mentioned that we'd want to support these eventually, analogous to `ExceptionPackage` for `exnref`, so the interpreter can run code by representing non-null `anyref` etc. at least abstractly.